### ISSUE TITLE
feature request: accept array of files, root directory

### ISSUE THREAD

**Issue by dylang** [kof/node-ams#4]
Order is important with js and css, so just scanning the directory often isn't helpful.

This is my workaround:
    var css = [
            'reset.css',
            'grid.css',
            'base.css',
            'jqueryui/jqueryui.css',
            'theme.css',
            'etc.css'
         ],
    js = [
            'jquery/jquery.js',
            'jquery/jquery-ui.js',
            'site.js',
            'plugins/jquery.fancybox-1.3.1.js',
            'plugins/dropdown.js',
            'plugins/toggleGrid.js'
        ];

```
css.forEach(function(filename) {
    ams.add('public/css/' + filename, 'public/css/');
});
js.forEach(function(filename) {
    ams.add('public/js/' + filename, 'public/js/');
});
```

I think it would be helpful if we could pass an array of files, something like:

```
ams.add(css, 'public/css/')
   .add(js, 'public/js/');
```

**Comment by kof** [comment]
I like your idea ... will be added.

**Comment by kof** [comment]
The reason why I was not caring about order is the intent of node-ams is to bring commonjs modules to the client and doing so you don't need to care about order, this is done by requirejs. I need to add more examples.

**Comment by kof** [comment]
While using ams you should'nt create arrays of files at all.

**Comment by kof** [comment]
Take a look at hello world example, it already shows how to use modules and requirejs.

**Comment by kof** [comment]
If you miss some informations I would like to know which, and would be glad to see any contribution in documentation, examples or code.

**Comment by kof** [comment]
Closed by d91206bd3972870c981b40e77d71cf1ec43c2e96. Build#add can now receive array of paths.
